browser-test/src/admin_program_creation.test.ts

```diff
@@ -19,6 +20,26 @@ describe('program creation', () => {
     await validateScreenshot(page, 'program-description-page')
   })
 
+  it('View active program, without draft and after creating draft', async () => {
```

Comment:
To explain my test strategy: This test covers all the new elements. I implemented it for the new flow only, because others tests go through the existing flow as a side effect already, but do not include the clicking of the view button for example.
In other words, this was the least effort way in which both the existing and the new flow get coverage.

In an ideal world the tests would automatically switch to the flag value that is used in staging or production and pass (e.g. if the flag is submitted). Based on our eng discussion this isn't currently supported. Enabling the flag will therefore require some changes to the tests.

Let me know in case you have come across a better approach or close this comment to acknowledge that you've read it and agree.

Reply:
I think you did the test as expected.  The tests shouldn't know or care about any environment, they should set the state they are testing and test that.

"In an ideal world the tests would automatically switch to the flag value that is used in staging or production and pass"

This doesn't line up with what we were trying to suggest, so sorry if there was confusion.

Something you should do though that I recently discovered, is explicitly disable the flag for any tests where it being on would make them break.   there's not `disableFeatureFlag` similar to `enableFeatureFlag` for that.  This will make the tests fully compatible when they're used as "probers" for environments with the feature enabled.

The easiest way to find those tests is to turn the flag on in your local `server/conf/application.dev-browser-tests.conf` and see which other tests fail and add the disable there.